test/extended/networking/livemigration.go

```diff
@@ -33,6 +33,8 @@ import (
 var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][Feature:Layer2LiveMigration] Kubevirt Virtual Machines", func() {
 	oc := exutil.NewCLIWithPodSecurityLevel("network-segmentation-e2e", admissionapi.LevelBaseline)
 	f := oc.KubeFramework()
+	// disable automatic namespace creation, we need to add the required UDN label
+	f.SkipNamespaceCreation = true
```

Comment:
`SkipNamespaceCreation` is already set to true here:  https://github.com/openshift/origin/blob/478335660e225b02f1f80aead7f50a2f85bad075/test/extended/util/client.go#L158

Reply:
done